Client<>AS<>IdP-OAPI.yaml

```diff
@@ -0,0 +1,343 @@
+openapi: 3.1.0
```

Comment:
Should this file be named `AS<>IdP-OAPI.yaml`?

Reply:
https://spec.openapis.org/oas/v3.1.0#document-structure
>It is RECOMMENDED that the root OpenAPI document be named: openapi.json or openapi.yaml.

Actually... what if these were in separate subdirectories?
Maybe something like (also changing the current [`open-api-spec.yaml`](https://github.com/interledger/open-payments/blob/main/open-api-spec.yaml))
```
openapi.yaml
AS/client/openapi.yaml
AS/IdP/openapi.yaml
AS/RS/openapi.yaml
```

Reply:
Even though this introduces a bunch of sub-directories, I like the structure.
Also, I love how you always dig up these things 👍

Reply:
Referencing relative yaml files across directories is starting to give me (codius) [kustomize](https://kustomize.io/) vibes
https://github.com/codius/codius-faas/blob/main/config/default/kustomization.yaml